src/main/java/chess/domain/piece/Knight.java

```diff
@@ -0,0 +1,58 @@
+package chess.domain.piece;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import chess.domain.Color;
+import chess.domain.position.Position;
+
+public class Knight extends Piece {
+
+    public Knight(Color color) {
+        super(color);
+    }
+
+    @Override
+    protected String baseSignature() {
+        return "n";
+    }
+
+    @Override
+    public boolean isMovable(Position source, Position target) {
+        int distanceX = Math.abs(source.calculateDisplacementXTo(target));
+        int distanceY = Math.abs(source.calculateDisplacementYTo(target));
+
+        if (distanceY == 1 && distanceX == 2) {
+            return true;
+        }
+        if (distanceY == 2 && distanceX == 1) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public List<Position> findRoute(Position source, Position target) {
+        return new ArrayList<>();
+    }
```

Comment:
사소한 이슈지만 새 객체를 만들어내지 않고, 
```suggestion
    @Override
    public List<Position> findRoute(Position source, Position target) {
        return List.of();
    }
```
만들어진 객체를 사용하는 것도 방법일 것 같네요~

Reply:
오 이런 방식으로 새로운 객체를 만들지 않아도 되는군요!! 좋은 정보 감사합니다 😀